src/client/components/pages/Courses/MeetingModal.tsx

```diff
@@ -37,37 +37,61 @@ interface MeetingModalProps {
  * Utility component to style content within meeting modal body
  */
 const MeetingModalBodyGrid = styled.div`
-  display: grid;
-  grid-template-columns: repeat(2, 1fr);
-  grid-template-rows: repeat(2, 1fr);
-  grid-template-areas:
-  "meetingScheduler roomAvailability"
-  "facultyNotes roomAvailability";
+  height: 75vh;
+  width: 75vw;
```

Comment:
I think these should be `max-height` and `max-width` instead of absolute values, so we don't end up with a modal that's significantly larger than its content.

Reply:
Good point! I replaced the `height` and `width` with `max-height` and `max-width.`